DESCRIPTION

```diff
@@ -159,7 +159,9 @@ Suggests:
     tgp,
     TH.data,
     wavelets,
-    xgboost (>= 0.6-2)
+    xgboost (>= 0.6-2),
+    sp,
+    gridExtra
```

Comment:
We try to keep the list of packages sorted alphabetically; please fix that.

Reply:
OK, I'll follow this rule